lib/apiservers/portlayer/swagger.json

```diff
@@ -1601,16 +1601,10 @@
 						}
 					},
 					"404": {
```

Comment:
I wonder why did you remove schema from the error? Does docker interpret error code only?